room.go

```diff
@@ -188,6 +192,10 @@ func NewRoom(callback *RoomCallback) *Room {
 		connectionState:    ConnectionStateDisconnected,
 		regionURLProvider:  newRegionURLProvider(),
 		rpcHandlers:        &sync.Map{},
```

Comment:
:nit: put this at the end to match declaration ☝️ . Makes it easier to read/eye-ball.